features/manage-position/Create.tsx

```diff
@@ -90,7 +90,7 @@ const Create = () => {
     collSymbol !== null &&
     priceIdentifier !== null
   ) {
-    const collReqFromWei = parseFloat(fromWei(collReq, collDec));
+    const collReqFromWei = parseFloat(fromWei(collReq, tokenDec));
```

Comment:
This is a mistake fixed throughout the PR: the collateral requirement unit;s should be the same as the token decimal (or maybe it should be 18) but it unintuitvely should NOT be the collateral's decimal